internal/actions/pr.go

```diff
@@ -137,6 +138,7 @@ func CreatePullRequest(ctx context.Context, repo *git.Repo, client *gh.Client, o
 	// figure this out based on whether or not we're on a stacked branch
 	branchMeta, _ := meta.ReadBranch(repo, opts.BranchName)
 	prBaseBranch := branchMeta.Parent.Name
+	localPRBaseBranch := prBaseBranch
```

Comment:
I think something like `prCompareRef` might be a better name for this.

Reply:
Done.